### PR TITLE
Teresa/feature/adding products

### PR DESCRIPTION
**Pull request by Saphala87**
Hi guys, check out all the coffee we got. 

We focused on making the pictures 640x640 
- but some of them might need some retouching later on (with a real photo-editor) :)

### CODE REVIEW

**Comment by Saphala87** [comment]
Ready for review!

File: mini-project-1/products.json [hunk 1]
```diff
@@ -2,20 +2,203 @@
   {
     "productId": 1,
     "productName": "Starbucks Espresso",
-    "price": 99,
+    "price": [59, 99, 129],
     "description": "Dark roasted beans for your delightful morning...",
-    "category": ["Dark roast", "SML"], //SML stands for available sizes
+    "category": ["Dark roast", "SML", "Starbucks"],
     "productImage": "assets/product_images/cafe1.png",
     "discount": "nodiscount",
```
Comment (Evgenios95) [inline mini-project-1/products.json:9]: What about changing this k-v pair from `"discount": "nodiscount"` to: `onDiscount: false`.
When we have only 2 options, it's better to keep it simple and use a boolean instead.

File: mini-project-1/products.json [hunk 2]
```diff
@@ -2,20 +2,203 @@
   {
     "productId": 1,
     "productName": "Starbucks Espresso",
-    "price": 99,
+    "price": [59, 99, 129],
     "description": "Dark roasted beans for your delightful morning...",
-    "category": ["Dark roast", "SML"], //SML stands for available sizes
+    "category": ["Dark roast", "SML", "Starbucks"],
```
Comment (Evgenios95) [inline mini-project-1/products.json:7]: I don't think we have a reason to have an array here. We know for example that this is a Starbucks espresso. 
So this value  should be `"category": "Espresso"`.
Then, we can use the filter method provided by JS to display only the products that belong under this category.
Same with all the other objects: `"category": "Medium Roast"`

File: mini-project-1/products.json [hunk 3]
```diff
@@ -2,20 +2,203 @@
...
     "discount": "nodiscount",
+    "rating": "5"
   },
   {
     "productId": 2,
     "productName": "Starbucks Holiday",
-    "price": 99,
-    "description": "Holiday stuff, good stuff...",
-    "category": ["Light roast", "SL"],
+    "price": [59, 129],
```
Comment (Evgenios95) [inline mini-project-1/products.json:15]: For the sake of project 1, I suggest removing the array of prices and use a single price instead.
Let's consider doing that later in React. But this is not even needed in the project description.
We need to filter by something, that's why we have our categories.

File: mini-project-1/products.json [hunk 4]
```diff
@@ -2,20 +2,203 @@
...
+    "description": "Enjoy our special blend of balanced and sweet flavours with herbal and maple notes made with the same high quality 100% Arabica beans.",
+    "category": ["Medium roast", "SL", "Starbucks"],
```
Comment (Evgenios95) [inline mini-project-1/products.json:17]: Remove the "SL" and all the other "SML". They are not a category.
Moreover there is a more elegant way to do it if we need to do it later.
We don't need to do that but consider some key value pairs like:

product_price: {
"S": 59,
"M": 79,
"L": 99,
}

**Review by Evgenios95 — CHANGES_REQUESTED** [review]
Looks nice in general, please discuss the comments that I left :)

**Pull request merged**